sr_description/hand/xacro/three_biotac_fingers_hand.urdf.xacro

```diff
@@ -10,11 +10,10 @@
   <xacro:macro name="shadowhand"
   params="muscletrans muscle bio bt_sp ubi eli reflect prefix">
     ${xacro.warning("The file three_biotac_fingers_hand.urdf.xacro is deprecated", print_location=False)}
-    ${xacro.warning("Use full_hand.urdf.xacro instead with tip_sensors=dict([(f, 'bio') for f in 'th ff mf'.split()])")}
+    ${xacro.warning("Use full_hand.urdf.xacro instead with tip_sensors='th=bio ff=bio mf=bio'")}
     <!-- forward to generic macro -->
     <xacro:include filename="$(find sr_description)/hand/xacro/full_hand.urdf.xacro" />
     <!-- BioTacs on First and Middle Finger as well as Thumb -->
-    <xacro:shadowhand reflect="${reflect}" prefix="${prefix}"
-    tip_sensors="${['bio', 'bio', 'bio', 'std', 'std']}"/>
+    <xacro:shadowhand reflect="${reflect}" prefix="${prefix}" tip_sensors="'th=bio ff=bio mf=bio'" />
```

Comment:
```suggestion
    <xacro:shadowhand reflect="${reflect}" prefix="${prefix}" tip_sensors="th=bio ff=bio mf=bio" />
```
to be similar to sr_description/robots/shadowhand_motor_th_ff_rf_ellipsoid.urdf.xacro which also does not need the extra single quotes. 
Careful, this macro is never instantiated (and was never in the past either, so will not be tested by the compatibility test right now). I did a manual test with some hand crafted instantiation in old and new form, and the result is fine.